src/analysis/calculus/deriv.lean

```diff
@@ -106,38 +106,50 @@ section derivative_uniqueness
 We prove that the definitions `unique_diff_within_at` and `unique_diff_on` indeed imply the
 uniqueness of the derivative. -/
 
+/-- If a function f has a derivative f', then the local behavior of f is expressed through f'. -/
```

Comment:
I have no idea what this docstring means. And I failed reading quickly the Lean statement.

Reply:
Clarified